site/content/docs/user/known-issues.md

```diff
@@ -198,7 +199,15 @@ at the kubelet logs, which may have something like the following:
 Dec 07 00:37:53 kind-1-control-plane kubelet[688]: I1207 00:37:53.229561     688 eviction_manager.go:340] eviction manager: must evict pod(s) to reclaim ephemeral-storage
 Dec 07 00:37:53 kind-1-control-plane kubelet[688]: E1207 00:37:53.229638     688 eviction_manager.go:351] eviction manager: eviction thresholds have been met, but no pods are active to evict
 ```
+## Failure to Create Cluster with Cgroups v2
+Host OS'es running Cgroups v2 are only supported for Kubernetes versions >= 1.19.
```

Comment:
Can we clarify that this is a kubernetes limitation (rather than kind specifically) and link to the release notes where kubernetes introduced support?

Reply:
Sure thing. PR updated.